### DISCUSSION TITLE
API Keys

### DISCUSSION THREAD

**Discussion by NickBraeckman** [lencx/ChatGPT#227] [Q&A] [answered]
Is there an option to connect to ChatGPT via an API key?

**Comment by lencx** [comment, marked as answer]
Support is not planned for the time being, as it is a bit of a huge amount of development and requires a display page to be created for it.

This is something I've been thinking about planning for a long time, and in the future it may be displayed and interacted with in a variety of ways, such as like a terminal command line, or just a plain page.

**Reply by laihenyi** [reply]
Paid-API RELEASE!

Will you add an option for users to use their API-key?
